src/main/java/utils/ExportResultsUtil.java

```diff
@@ -1,50 +1,49 @@
 package utils;
 
 import refactoring.Refactoring;
+
 import javax.swing.*;
-import java.io.File;
+import java.awt.*;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
+
 import static java.nio.file.Files.createFile;
 import static java.nio.file.Files.write;
 
 /**
- * Class exports (MoveMethod) refactoring results to the file
+ * Exports refactoring results to the file
  */
 public class ExportResultsUtil {
```

Comment:
Please, see `com.intellij.ide.util.ExportToFileUtil`
Can we use `ExportToFileUtil`?